tests/test_rain.py

```diff
@@ -7,26 +7,28 @@
 
 from meteofrance_api import MeteoFranceClient
 from meteofrance_api.const import METEOFRANCE_API_URL
+from meteofrance_api.model import Rain
 
 
 def test_rain() -> None:
     """Test rain forecast on a covered zone."""
     client = MeteoFranceClient()
 
     rain = client.get_rain(latitude=48.8075, longitude=2.24028)
-
-    assert type(rain.position) == dict
-    assert type(rain.updated_on) == int
-    assert type(rain.quality) == int
+    assert isinstance(rain, Rain)
+    assert isinstance(rain.position, dict)
+    assert isinstance(rain.updated_on, int)
+    assert isinstance(rain.quality, int)
```

Comment:
Changing type to isinstance could be done everywhere in one separate PR, same for #680